pkg/bootstrap/bootstrap_config.go

```diff
@@ -306,7 +377,7 @@ func WriteBootstrap(config *meshconfig.ProxyConfig, node string, epoch int, pilo
 	// Support passing extra info from node environment as metadata
 	meta := getNodeMetaData(localEnv)
 
-	localityOverride := model.GetLocalityOrDefault("", meta)
+	localityOverride := model.GetLocalityOrDefault("", stringMap(meta))
```

Comment:
may be simpler to just change the locality method to take in the `LocalityLabel` directly instead of the full set of labels -- not sure why it took all the labels in the first place. Then we can get rid of the `stringMap`

Reply:
OK. I updated `GetLocalityOrDefault` to take a string. As part of that refactor, I switched the order of the args to be `label, default` to be more inline with convention around getOrDefault methods. I've updated the calling sites to match.